### PR TITLE
Update to Gradle 7

### PR DESCRIPTION
**Pull request by jsarabia**
This change is [Reviewable](https://reviewable.io/reviews/Bible-Translation-Tools/Orature/746)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -73,18 +73,18 @@ tasks.withType(io.gitlab.arturbosch.detekt.Detekt) {
...
+//    license {
+//        ignoreFailures = true
+//        header = rootProject.file("LICENSE")
+//        mapping {
+//            gradle = 'JAVADOC_STYLE'
+//            fxml = 'XML_STYLE'
+//        }
+//        includes(["**/*.kt", "**/*.java", "**/*.properties", "**/*.css", "**/*.fxml", "**/*.gradle"])
+//        excludes(["**/src/main/java/jooq/**/*.java", "**/*.sqlite"])
+//    }
```
Comment (AnonymousWalker) [inline build.gradle]: how should we go about this task?

Comment (AnonymousWalker) [inline build.gradle:87]: how should we go about this task?

Comment (jsarabia) [reply build.gradle]: It didn't seem to be working previously but it seems to be okay now, so maybe it wasn't pulling the latest version previously

**Review by AnonymousWalker — APPROVED** [review]

**Comment by AnonymousWalker** [comment]
the build failed again, somehow :/

**Pull request merged**